source/ed/build_helper.cpp

```diff
@@ -820,16 +800,16 @@ void builder::finish() {
 }
 
 void builder::make() {
-    generate_dummy_basis();
```

Comment:
you don't need to generate_dummy_basis() anymore ?

Reply:
We still do indeed :)  
As a lot of tests need the dummy objects prior to the `make()` call,  I moved the call to `generate_dummy_basis()` [just before the callback](https://github.com/CanalTP/navitia/pull/3388/files#diff-f965e99255fc0119c323b338b3eadd33032497f5d44354cabb7061f98d4c6a9dR632-R633) in ed::builder's constructor.

Reply:
allright :+1: